spec/presenters/pagination_presenter_spec.rb

```diff
@@ -51,8 +51,8 @@
       expect(subject.next_link?).to eq(false)
     end
 
-    it 'returns the correct next page label' do
-      expect(subject.prev_label).to eq('123456 of 123457')
+    it 'returns the correct previous page label' do
+      expect(subject.prev_label).to eq('123,456 of 123,457')
     end
 
     it 'returns 1234561 for #first_record' do
```

Comment:
update the expectation with a comma to reflect what you're actually expecting

Reply:
This is accurate - `first_record` method return an integer and not the formatted string

Reply:
My mistake, misread the diff, thought the inner didn't match the outer on the test.